src/lib.js

```diff
@@ -238,14 +237,14 @@ async function publishBundle(bundle, handlers, outputDir, isLocal, apiHost, apiT
     await makeDir(outputDir)
 
     // bundled handlers
-    const outputFile = path.join(outputDir, bundleInfo.sha)
```

Comment:
`path.join()` is not needed when using core `fs.*()` methods or libraries using those methods (like `del` and `make-dir`) because [those core methods handle Unix file paths even on Windows](https://github.com/ehmicky/cross-platform-node-guide/blob/main/docs/3_filesystem/file_paths.md#nodejs-behavior).